README.md

```
@@ -56,3 +56,19 @@ Video of Configuring Photo Collage Screensaver: https://www.youtube.com/watch?v=
 ### Testimonials
 
 > This is a fantastic screensaver and the borders add an artistic touch that no other product comes close to. - Doug H.
+
+## Development 
+
+**Publishing**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)